drivers/hmis/app/graphql/types/application/user.rb

```diff
@@ -9,6 +9,7 @@
 module Types
   class Application::User < Types::BaseObject
     include Types::HmisSchema::HasAuditHistory
+    include Hmis::Concerns::HmisArelHelper
```

Comment:
This is fine, we do this in lots of places in the application.

However I think it would be better to use composition if possible. Suggest adding an arel helper to `GraphqlApplicationHelper`
```
  def arel
    Hmis::ArelHelper.instance
  end
```

Then adjust how we call these arel table aliases: s/hh_t/arel.hh_t/

Reply:
Thanks for the suggestion Theron, can you elaborate more on why this is better - is it for clarity of code or for another reason? In any case, I'm happy to implement your suggestion here if that's the pattern we should standardize around going forward.

https://github.com/greenriver/hmis-warehouse/pull/4578/commits/542ea94671f2108e5b6ce4c352ac3218fe4e41a6

Reply:
Thanks Martha, good question! In this case, with our arel helpers, we just want access to a set of constants that define table names. By using an arel helper singleton, we avoid polluting the caller's namespace with these constants, reducing the changes of name collision (of particular concern with short, non-descriptive vars like 'hh_t')

More generally there's something of an adage in OOP: "favor composition over inheritance". It's not super relevant here but the idea is to encourage designs that compose behavior using smaller, loosely coupled components rather than mixing behavior together into one big class using inheritance or modules. Of course this is just a guideline, modules are frequently the most pragmatic approach